apps/frontend/src/app/Data/Characters/Bennett/index.tsx

```diff
@@ -90,34 +91,32 @@ const dm = {
   },
 } as const
 
-const a1SkillCd = greaterEq(input.asc, 1, dm.passive1.cd_red)
-
 const burstAtkRatio = subscript(input.total.burstIndex, dm.burst.atkBonus, {
   unit: '%',
 })
 const burstAddlAtk = prod(burstAtkRatio, input.base.atk)
-const c1AtkRatio = greaterEq(
-  input.constellation,
-  1,
-  dm.constellation1.atk_inc,
-  { name: ct.ch('additionalATKRatio_'), unit: '%' }
-)
-const c1AddlAtk = greaterEq(
-  input.constellation,
-  1,
-  prod(c1AtkRatio, input.base.atk)
+const c1AddlAtk = infoMut(
+  greaterEq(
+    input.constellation,
+    1,
+    prod(percent(dm.constellation1.atk_inc), input.base.atk)
+  ),
+  { name: ct.ch('additionalATK') }
 )
-const atkIncRatio = sum(burstAtkRatio, c1AtkRatio)
-const activeInAreaAtkDisp = prod(atkIncRatio, input.base.atk)
-
 const [condInAreaPath, condInArea] = cond(key, 'activeInArea')
 const activeInArea = equal(
   'activeInArea',
   condInArea,
   equal(input.activeCharKey, target.charKey, 1)
 )
+const activeInAreaAtkDisp = {
+  ...sum(burstAddlAtk, c1AddlAtk),
```

Comment:
We separated these on purpose because the UI is confusing when combined. The C1 header indicates it is adding more atk, but its actually already factored into the calculation

Reply:
So what I did was change the C1 listing to instead talk about the buff it gives. I think this should be clear enough.

It's gonna display like this: 
![image](https://github.com/frzyc/genshin-optimizer/assets/121315943/571a9226-6e31-43fa-9384-36f9f2da3b40)
tell me what you think :3